interop/src/main.rs

```diff
@@ -69,7 +72,10 @@ fn run(log: Logger, options: Opt) -> Result<()> {
     tls_config
         .dangerous()
         .set_certificate_verifier(Arc::new(InteropVerifier(state.clone())));
-    tls_config.alpn_protocols = vec![str::from_utf8(quinn::ALPN_QUIC_HTTP).unwrap().into()];
+    tls_config.alpn_protocols = vec![
+        str::from_utf8(quinn::ALPN_QUIC_HTTP).unwrap().into(),
+        str::from_utf8(quinn::ALPN_QUIC_H3).unwrap().into(),
+    ];
```

Comment:
We should have two separate client configs, one for `hq-whatever` and one for `h3-whatever`. Placing both in a single config doesn't mean you can use either, it means you're willing to let the server select either for you, and it's your responsibility to comply with the server's selection. Plus, this way you can enable/disable incoming unidirectional streams for each as appropriate.

Also, we can get rid of the `str::from_utf8(...).unwrap()` now.

Reply:
fixed in 4ee65b9